clients/e-shop-demo/e-shop-frontend/src/data/tooltips.ts

```diff
@@ -48,7 +48,7 @@ export const steps: Step[] = [
 	{
 		title: 'Checkout',
 		target: '.credentialAgeRestriction',
-		content: 'The credential proofs that you are under the age of 18 and not allowed to buy liquor. Try the adult credential.',
+		content: 'The credential proofs that you are under the age of 18 or submitted an invalid credential and are not allowed to buy liquor. Try the adult credential.',
```

Comment:
Maybe adjust the text so it is easier to read.

The credential proofs that you are under the age of 18 or submitted an invalid credential. In this case you are not allowed to buy liquor. You can try the adult credential to proceed the checkout.